storage/SystemInfo/CmdDmraid.cc

```diff
@@ -80,22 +81,22 @@ namespace storage
 
 	for (const_iterator it = data.begin(); it != data.end(); ++it)
 	    y2mil("data[" << it->first << "] -> controller:" << it->second.controller <<
-		  " raidtype:" << it->second.raidtype << " devices:" << it->second.devices);
+		  " raid-type:" << it->second.raid_type << " devices:" << it->second.devices);
     }
 
 
-    list<string>
-    CmdDmraid::getEntries() const
+    vector<string>
+    CmdDmraid::get_entries() const
     {
-	list<string> ret;
+	vector<string> ret;
 	for (const_iterator it = data.begin(); it != data.end(); ++it)
 	    ret.push_back(it->first);
 	return ret;
     }
 
 
     bool
-    CmdDmraid::getEntry(const string& name, Entry& entry) const
+    CmdDmraid::get_entry(const string& name, Entry& entry) const
```

Comment:
Hm... IMHO a `get_xy()` function should return the xy as its return value, not a boolean ok and as a byproduct modify a non-const reference.

Alternative approaches:

- `Entry CmdDmraid::get_entry( const string & name ) const
  that throws an exception if not found - this would probably be appropriate in this case since not getting a valid result for a name (that you fetched from some other place here) is most likely a programmatic error.

- `Entry * get_entry( const string & name ) const
  that returns a null pointer if not found.

But always returning the less important ok flag and having to do mental acrobatics for that is not the best solution IMHO.